### ISSUE TITLE
question about  "-1" and "0"

### ISSUE THREAD

**Issue by zml110120** [232525/PureT#9]
Dear author,
I have a question.

What is the difference between "-1" in "TARGET_SENT" and "0" in "INPUT_SENT"？ Just like the first image and second image? "-1" means padding and "0" means the end of sentence?

![image](https://user-images.githubusercontent.com/29172684/182608485-34e54895-090d-4b0e-91d6-cb936dd52493.png)
![image](https://user-images.githubusercontent.com/29172684/182609661-b50a146f-8c64-468e-b808-e226b4289128.png)

**Comment by 232525** [comment]
> Dear author, I have a question.
> 
> What is the difference between "-1" in "TARGET_SENT" and "0" in "INPUT_SENT"？ Just like the first image and second image? "-1" means padding and "0" means the end of sentence?
> 
> ![image](https://user-images.githubusercontent.com/29172684/182608485-34e54895-090d-4b0e-91d6-cb936dd52493.png) ![image](https://user-images.githubusercontent.com/29172684/182609661-b50a146f-8c64-468e-b808-e226b4289128.png)

Right! And: "-1" in "TARGET_SENT" is convenient for computing loss.
+ XE loss
<img width="517" alt="image" src="https://user-images.githubusercontent.com/20409609/182614003-860f0dcd-574c-41d7-a8e9-2d1c1f01cacf.png">

+ label smoothing
<img width="639" alt="image" src="https://user-images.githubusercontent.com/20409609/182614058-0e8f800a-c549-40b3-9a96-0f430904edb6.png">
